.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
```

Comment:
These changes should be in a separate commit (and rationale should be provided through comments).

Reply:
ok thanks for the reply. Should I go ahead to revert the commit that contains it? or manually remove the changes in a new commit?

Reply:
Whichever you prefer. We'll squash this PR anyways.

Couldn't you install this "something" outside of this directory? (Also, you can just stage selected files for committing. That's what I usually do.)

Reply:
okay thanks for the tip. I'll do that